src/searchForm/datePicker/datePickerTopBar/DatePickerTopBar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import CalendarIcon from 'icon/calendarIcon'
+import Text from 'text/Text'
+import Button, { ButtonStatus } from 'button'
+import cc from 'classcat'
+import ChevronIcon from 'icon/chevronIcon'
+
+export interface DatePickerTopBarProps {
```

Comment:
I don't see the value of having this widget, we have two completely different behavior/rendering merged into one place. I would just dispatch each top bars in overlay and fullscreen tsx